### PR TITLE
Add slf4j-simple to examples

### PR DESCRIPTION
**Pull request by okue**
Without it, no outputs...

```
> Task :examples:context-propagation-dagger-example:run
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
<============-> 95% EXECUTING [3m 23s]
> IDLE
> :examples:context-propagation-dagger-example:run
```

### CODE REVIEW

File: examples/context-propagation/kotlin/build.gradle.kts [hunk 1]
```diff
@@ -5,12 +5,13 @@ plugins {
...
+    runtimeOnly("org.slf4j:slf4j-simple")
 
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8")
 }
 
 application {
-    mainClassName = "example.armeria.contextpropagation.kotlin.Main"
+    mainClassName = "example.armeria.contextpropagation.kotlin.MainKt"
```
Comment (ikhoon) [inline examples/context-propagation/kotlin/build.gradle.kts:16]: Oh... I didn't know that `Main.kt` is compiled to `MainKt.class`.

**Comment by minwoox** [comment]
How about adding to other examples as well?

**Comment by minwoox** [comment]
Or we could just add it to https://github.com/line/armeria-examples/blob/master/update-examples.sh#L204
so that all examples have the dependency. 🤔

**Comment by okue** [comment]
> How about adding to other examples as well?

All projects have logger dependency as spring-boot depends on `logback`.

**Comment by okue** [comment]
Aa, isn't `examples` in this repository for users?
Is `armeria-examples` for users?

**Comment by okue** [comment]
I often tried running examples in this repo 😅

**Comment by minwoox** [comment]
> Is armeria-examples for users?

Yes. 😅 
We use the shell script to fill the missing dependencies in build.gradle.
https://github.com/line/armeria-examples/blob/master/update-examples.sh

**Comment by minwoox** [comment]
I guess all examples had spring-boot dependency or slfj4 directly except these propagation examples. 😆 
Thanks a lot for fixing this, @okue!

**Pull request merged**